src/main/kotlin/no/skatteetaten/aurora/boober/feature/ToxiproxySidecarFeature.kt

```diff
@@ -31,22 +31,30 @@ import no.skatteetaten.aurora.boober.model.PortNumbers
 import no.skatteetaten.aurora.boober.model.findConfigFieldHandlers
 import no.skatteetaten.aurora.boober.model.findSubKeysExpanded
 import no.skatteetaten.aurora.boober.service.CantusService
+import no.skatteetaten.aurora.boober.service.UserDetailsProvider
+import no.skatteetaten.aurora.boober.service.resourceprovisioning.DatabaseSchemaProvisioner
 import no.skatteetaten.aurora.boober.utils.allNonSideCarContainers
 import no.skatteetaten.aurora.boober.utils.boolean
 import no.skatteetaten.aurora.boober.utils.notBlank
 import no.skatteetaten.aurora.boober.utils.prependIfNotNull
-import org.springframework.beans.factory.annotation.Value
-
-private const val FIRST_PORT_NUMBER = 18000 // The first Toxiproxy port will be set to this number
 
 private const val TOXIPROXY_CONFIGS_CONTEXT_KEY = "toxiproxyConfigs"
 
 val FeatureContext.toxiproxyConfigs: List<ToxiproxyConfig>
-    get() = this.getContextKey(TOXIPROXY_CONFIGS_CONTEXT_KEY)
+    get() = getContextKey(TOXIPROXY_CONFIGS_CONTEXT_KEY)
+
+private const val SECRET_NAME_TO_PORT_MAP_CONTEXT_KEY = "secretNameToPortMap"
+
+private const val FEATURE_NAME = "toxiproxy"
+
+private val FeatureContext.secretNameToPortMap: Map<String, Int>
+    get() = getContextKey(SECRET_NAME_TO_PORT_MAP_CONTEXT_KEY)
 
 @org.springframework.stereotype.Service
 class ToxiproxySidecarFeature(
```

Comment:
Har noen tatt stilling til om denne bør vurdere være tilgjengelig i relay? 
Database er ikke tilgjengelig i relay. Bør det være da lov til å spesifisere toxiproxy database for relay?